### DISCUSSION TITLE
Problem running 2.0 on Windows 10

### DISCUSSION THREAD

**Discussion by Nodling** [edwardkort/WWIDesigner#70] [Setup]
I have recently installed the 2.0 version (and  the most recent version of JAVA) and have been following your excellent tutorials on Youtube.  The program opens apparently normally but when I try to load a study, I get a series of error messages in the console.  The first line is: Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefounderror: javaz/xml/bind/JaxBContext.

Any suggestions as tyo how I might get around this?

**Comment by edwardkort** [comment]
javax.xml.bind.JaxBContext (not javaz) is part of the java rt.jar (the major java run-time library). So either you have a hacked copy of the WIDesigner zip file, or you have a problem with your java installation.

Download the WIDesigner 2.1.0 zip file and try it. Download a checksum utility and make sure that your downloaded zip has the same checksum as in the "Checksums-2.1.0.txt.

There are several java distributions out there. Make sure that you are running the one from www.oracle.com. The jre installed on my windows 10 box is 1.8.0_161. However, I run my java apps with the jdk; I'm still using version 8.0_66 for that. Nonetheless, verify that your java is working with another application.

**Comment by edwardkort** [comment]
No confirmation of issue. Closing.
